Casks/d/double-commander.rb

```diff
@@ -1,22 +1,19 @@
 cask "double-commander" do
   arch arm: "aarch64", intel: "x86_64"
 
-  version "1.1.8,247"
-  sha256 arm:   "69d88f6f1f4565295c8d8218b4170a515277d5f2c3a6027d2178011bf188f886",
-         intel: "282b272a0ee56519de875a7d539f734a4490479c0dfc2681919eb6ca19df3e04"
+  version "1.1.10"
+  sha256 arm:   "4f6fe3b2023c0cc123a5caba6fce27be63f3023b3e9aca91ea2f668659cbd691",
+         intel: "3a827eb57095071de16b85a1054e053cceec2ea448c0c1430f43396a0089c9be"
 
-  url "https://downloads.sourceforge.net/doublecmd/doublecmd-#{version.csv.first}-#{version.csv.second}.cocoa.#{arch}.dmg",
+  url "https://downloads.sourceforge.net/doublecmd/doublecmd-#{version}.cocoa.#{arch}.dmg",
       verified: "downloads.sourceforge.net/doublecmd/"
   name "Double Commander"
   desc "File manager with two panels"
   homepage "https://doublecmd.sourceforge.io/"
 
   livecheck do
     url "https://sourceforge.net/projects/doublecmd/rss?path=/macOS"
-    regex(%r{url=.*?/doublecmd[._-](\d+(?:\.\d+)+)[._-](\d+)[^"' ]*?\.dmg}i)
-    strategy :page_match do |page, regex|
-      page.scan(regex).map { |match| "#{match[0]},#{match[1]}" }
-    end
+    regex(%r{url=["']?.*?/doublecmd[._-](\d+(?:\.\d+)+)[._-][^"']*?.dmg}i)
```

Comment:
```suggestion
    regex(%r{url=.*?/doublecmd[._-](\d+(?:[.-]\d+)+)[^"' ]*?\.dmg}i)
    strategy :sourceforge do |page, regex|
      page.scan(regex).map { |match| match[0].tr("-", ",") }
    end
```

The newest two versions (1.1.9, 1.1.10) don't include a numeric suffix like the previous versions (e.g., 1.1.8-247) but versions as recent as 2023-12-16 include a numeric suffix. The current regex in this PR would omit any numeric suffix from the from the version and it would return a version like 1.1.8 even if the full version in the filename is 1.1.8-247, which would make this harder to version bump.

There's something to be said for supporting both formats in the `livecheck` block, so it will surface versions like 1.1.10 and 1.1.8-247.

Reply:
done, thx